google/cloud/aiplatform/metadata/metadata.py

```diff
@@ -94,11 +119,78 @@ def log_metrics(self, metrics: Dict[str, Union[str, float, int]]):
         )
         artifact.update(metadata=metrics)
 
-    def get_experiment(self, experiment: str):
-        raise NotImplementedError("get_experiment not implemented")
+    def get_experiment(self, experiment: str) -> pd.DataFrame:
+        """Returns a Pandas DataFrame of the parameters and metrics associated with one experiment.
+
+            Example:
+
+            aiplatform.init(experiment='exp-1')
+            aiplatform.set_run(run='run-1')
+            aiplatform.log_params({'learning_rate': 0.1})
+            aiplatform.log_metrics({'accuracy': 0.9})
+
+            aiplatform.set_run(run='run-2')
+            aiplatform.log_params({'learning_rate': 0.2})
+            aiplatform.log_metrics({'accuracy': 0.95})
+
+            Will result in the following DataFrame
+            ___________________________________________________________________________
+            | experiment_name | run_name      | param.learning_rate | metric.accuracy |
```

Comment:
do we need the experiment_name column?

Reply:
I think having the experiment_name in the df would help user for their analysis. Some use case I can imaging would be user might want to pull a couple different experiment, or pipeline data and then append them together for some comparison(ex, maybe compare recent experiment with previous ones). And in those cases, the experiment_name column would help them distinguish the data.

Reply:
SG